### DISCUSSION TITLE
Can't convert JPG, colors are f*ed up afterwards

### DISCUSSION THREAD

**Discussion by Twisterking** [ImageMagick/ImageMagick#3897] [Help]
Hi there,

I really need help here! Until now I used to following command to resize and optimize images before I upload and use them on the web: `mogrify -strip -quality 85 -fuzz %1 -trim +repage -background White -alpha remove -format jpg -resize '1280x1024>' -path ./large "*.*"`

This suddenly stopped working for some JPG images, e.g. this one: 
![before](https://user-images.githubusercontent.com/4247268/124478951-03068280-dda6-11eb-96d5-b43eb02a00f5.jpg)
After running the command, the following image is saved: 
![after](https://user-images.githubusercontent.com/4247268/124479019-19144300-dda6-11eb-8558-f06c161d81b2.jpg)

Can someone help me please? I am completely lost here - what am I doing wrong?

Thanks, best,
Patrick

**Comment by snibgo** [comment]
What version of IM, on what platform?

As you can see, the tones and colours have negated. This suggests a CMYK/RGB problem. Sure enough, your image is CMYK with an embedded ICC profile.

The profile defines the colorimetry of the channels, and hence the colour that should be shown at each pixel. You "-strip" the profile, so the definition is lost. Instead, I suggest you use "-profile sRGB.icc", using the full path to wherever sRGB.icc is on your computer. (It is installed with ImageMagick.) Or use any sRGB profile.
